README.md

```diff
@@ -20,7 +20,7 @@
   <p align="center">
     A Place for All Your Recipes
     <br />
-    <a href="https://nightly.mealie.io"><strong>Explore the docs »</strong></a>
+    <a href="https://docs.mealie.io/documentation/getting-started/introduction/"><strong>Explore the docs »</strong></a>
```

Comment:
I'm fine with changing `nightly` -> `docs`, but I'm not sure it's correct to be adding all the other stuff there. Do you have a rationale behind it?

Reply:
My reasoning for directly linking to that is because that's the first page of the actual docs, going to just docs.mealie.io takes you to the base page of docs which has no content of the docs themselves from what I can tell and you have to press getting started to go to the docs, so it just saves an extra click

Reply:
So my concern with that is we're stating that we're linking to the docs, and that's what we're doing. Changing it to link to the getting started guide is well, not _the docs_ but _the getting started guide_.
I don't think we need to be/should be adding `/documentation/getting-started/introduction/`

Reply:
I was in under the impression that the requested change is the docs while just being worded as "getting started" otherwise I cannot find any other documentation. The existing existing link (nightly.mealie.io) simply redirects to docs.mealie.io so why use this even in the first place? The new link also directly takes you to the 'docs' rather than the menu to get to the docs. Personally, when I click on a link to go to somewhere like docs I expect it to directly go there not a page that redirects me and then allows me to find it and then go there.

Reply:
I agree with @boc-the-git, I don't see a reason for this change. It makes sense to me to change `nightly` to `docs` but I don't think it makes sense to take them to a subpage rather than the homepage

Reply:
I understand, perhaps it's better to have the explore the docs as docs.mealie.io and the documentation link to go to the docs.mealie.io/documentation/getting-started/introduction/ ?

Reply:
I don't see a reason to change either link, besides switching from `nightly` to `docs`

Reply:
Sounds good, will change it to docs